.github/workflows/pre-commit.yaml

```diff
@@ -1,16 +1,18 @@
-name: Pre-commit on PR
+name: Pre-Commit
 on:
   pull_request:
   push:
-    branches: [main, test-me-*]
-
+    branches: [main, master]
```

Comment:
configurable branches will be good, just have this values as default
here and following all places